### PR TITLE
Home task:HTML 02

### PR DESCRIPTION
**Pull request by Vecherskiys**
Please review.
Thank you!

### CODE REVIEW

File: Vecherskiys/02/Task2.1.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<HTML>
+ <HEAD>
+ <TITLE>Месяц моего рождения</TITLE>
```
Comment (pluhin) [inline Vecherskiys/02/Task2.1.html]: Where is meta tag?

File: Vecherskiys/02/Task2.1.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ </HEAD>  
+  <BODY> 
+  <TABLE border="1"; width="210"; align="center">
```
Comment (pluhin) [inline Vecherskiys/02/Task2.1.html]: could you please remove ;

File: Vecherskiys/02/Task2.2v1.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<HTML>
+ <HEAD>
+ <TITLE>Прогноз погоды</TITLE>
```
Comment (pluhin) [inline Vecherskiys/02/Task2.2v1.html]: where is meta tag?

File: Vecherskiys/02/Task2.2v1.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+ </HEAD>  
+  <BODY> 
+  <TABLE border="1"; width="500"; align="center">
```
Comment (pluhin) [inline Vecherskiys/02/Task2.2v1.html]: remove ;

File: Vecherskiys/02/Task2.2v2.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<HTML>
+ <HEAD>
+ <TITLE>Прогноз погоды</TITLE>
```
Comment (pluhin) [inline Vecherskiys/02/Task2.2v2.html]: missing meta tag

File: Vecherskiys/02/Task2.2v2.html [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+ </HEAD>  
+  <BODY> 
+  <TABLE border="1"; width="500"; align="center">
```
Comment (pluhin) [inline Vecherskiys/02/Task2.2v2.html]: ; - is extra

File: Vecherskiys/02/pic/Python.add/Task2.3.html [hunk 7]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<HTML>
+ <HEAD>
+ <TITLE>HOW-TO: Программа на Python, Часть 1</TITLE>
```
Comment (pluhin) [inline Vecherskiys/02/pic/Python.add/Task2.3.html]: missing meta

File: Vecherskiys/02/Task2.1.html (hunk 2 above)
Comment (Vecherskiys) [reply Vecherskiys/02/Task2.1.html]: Ok

File: Vecherskiys/02/Task2.2v1.html (hunk 4 above)
Comment (Vecherskiys) [reply Vecherskiys/02/Task2.2v1.html]: Ok

File: Vecherskiys/02/Task2.2v2.html (hunk 6 above)
Comment (Vecherskiys) [reply Vecherskiys/02/Task2.2v2.html]: Ok)

File: Vecherskiys/02/Task2.1.html (hunk 1 above)
Comment (Vecherskiys) [reply Vecherskiys/02/Task2.1.html]: Fixed

File: Vecherskiys/02/Task2.2v2.html (hunk 5 above)
Comment (Vecherskiys) [reply Vecherskiys/02/Task2.2v2.html]: Fixed

File: Vecherskiys/02/Task2.1.html [hunk 8]
```diff
@@ -1,10 +1,11 @@
 <!DOCTYPE html>
 <HTML>
  <HEAD>
+ <meta name="Description" content="My MHB">
```
Comment (pluhin) [inline Vecherskiys/02/Task2.1.html]: meta tag for charset https://www.w3schools.com/tags/att_meta_charset.asp

File: Vecherskiys/02/Task2.2v1.html [hunk 9]
```diff
@@ -1,10 +1,11 @@
 <!DOCTYPE html>
 <HTML>
  <HEAD>
+ <meta name="Description" content="Weather">
```
Comment (pluhin) [inline Vecherskiys/02/Task2.2v1.html]: meta tag for charset https://www.w3schools.com/tags/att_meta_charset.asp

File: Vecherskiys/02/Task2.2v2.html [hunk 10]
```diff
@@ -1,10 +1,11 @@
 <!DOCTYPE html>
 <HTML>
  <HEAD>
+ <meta name="Description" content="Weather2">
```
Comment (pluhin) [inline Vecherskiys/02/Task2.2v2.html]: meta tag for charset https://www.w3schools.com/tags/att_meta_charset.asp

File: Vecherskiys/02/pic/Python.add/Task2.3.html [hunk 11]
```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <HTML>
  <HEAD>
+ <meta name="Description" content="About Python">
```
Comment (pluhin) [inline Vecherskiys/02/pic/Python.add/Task2.3.html]: meta tag for charset https://www.w3schools.com/tags/att_meta_charset.asp

File: Vecherskiys/02/pic/Python.add/Task2.5.html [hunk 12]
```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <HTML>
  <HEAD>
+ <meta name="Description" content="About Python">
```
Comment (pluhin) [inline Vecherskiys/02/pic/Python.add/Task2.5.html]: meta tag for charset https://www.w3schools.com/tags/att_meta_charset.asp

File: Vecherskiys/02/pic/Python.add/Task2.4.html [hunk 13]
```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <HTML>
  <HEAD>
+ <meta name="Description" content="About Python">
```
Comment (pluhin) [inline Vecherskiys/02/pic/Python.add/Task2.4.html]: meta tag for charset https://www.w3schools.com/tags/att_meta_charset.asp

File: Vecherskiys/02/Task2.1.html (hunk 8 above)
Comment (Vecherskiys) [reply Vecherskiys/02/Task2.1.html]: Ok

File: Vecherskiys/02/Task2.2v1.html (hunk 9 above)
Comment (Vecherskiys) [reply Vecherskiys/02/Task2.2v1.html]: Ok

File: Vecherskiys/02/Task2.2v2.html (hunk 10 above)
Comment (Vecherskiys) [reply Vecherskiys/02/Task2.2v2.html]: Ok

File: Vecherskiys/02/pic/Python.add/Task2.3.html (hunk 11 above)
Comment (Vecherskiys) [reply Vecherskiys/02/pic/Python.add/Task2.3.html]: Ok

File: Vecherskiys/02/pic/Python.add/Task2.5.html (hunk 12 above)
Comment (Vecherskiys) [reply Vecherskiys/02/pic/Python.add/Task2.5.html]: Ok

File: Vecherskiys/02/pic/Python.add/Task2.4.html (hunk 13 above)
Comment (Vecherskiys) [reply Vecherskiys/02/pic/Python.add/Task2.4.html]: Ok

**Pull request merged**